recipe/meta.yaml

```diff
@@ -11,7 +11,7 @@ source:
   sha256: b6a73b1b7d2ee5305f919d9f94883abc68e8c948257e22ce8771a72a22b46d95
 
 build:
-  number: 0
+  number: 1
```

Comment:
Do we need new 2.19.1 binaries built and uploaded with this updated dependency?

If this fix is primarily in preparation for the next release of TileDB, I think we should merge the recipe update (to fix our nightlies and be ready for the next PR from the auto-tick bot) without bumping the build number (to avoid uploading redundant 2.19.1 binaries to anaconda.org)

Reply:
I'd also like to avoid having libboost before its needed.

Reply:
Hmm, the boost dependency will be added in 2.21. Is there a way to add it only in nightly builds?